docs/http_api.rst

```diff
@@ -40,6 +40,13 @@ Channels
        to. May be ``null`` if not desired. Not supported by every channel.
    :param str mo_url:
        URL to call on incoming messages (mobile originated) from this channel.
+       One or both of mo_url or amqp_queue must be specified.
```

Comment:
Should we perhaps mention that if both are specified, MO messages are sent to both?